File: Easy/2026-01-25-1984-Minimum-Difference-Between-Highest-and-Lowest-of-K-Scores/solution.js
var minimumDifference = function(nums, k) {
    nums.sort((a, b) => a - b);
    let minDiff = Infinity;

    for (let i = 0; i <= nums.length - k; i++) {
        const diff = nums[i + k - 1] - nums[i];
        minDiff = Math.min(minDiff, diff);
    }

    return minDiff;
};

File: Easy/2026-01-25-1984-Minimum-Difference-Between-Highest-and-Lowest-of-K-Scores/solution.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        nums.sort()
        min_diff = float('inf')
        
        for i in range(len(nums) - k + 1):
            current_diff = nums[i + k - 1] - nums[i]
            min_diff = min(min_diff, current_diff)
        
        return min_diff